discord/enums.py

```diff
@@ -172,6 +173,11 @@ class MessageType(Enum):
     premium_guild_tier_2       = 10
     premium_guild_tier_3       = 11
 
+class MessageFlags(Enum):
+    crossposted    = 1
+    is_crosspost   = 2
+    suppress_embeds = 4
```

Comment:
1,2,4? Is it intended? Shouldn't suppress_embeds's value be 3?

Reply:
Intended. See: https://discordapp.com/developers/docs/resources/channel#message-object-message-flags

Much like UserFlags.